analyses/sample-distribution-analysis/00-install-packages.R

```diff
@@ -0,0 +1,57 @@
+# Install required packages
+if (!require(dplyr)) {
+  install.packages("dplyr", repos = 'http://cran.us.r-project.org')
+  library(dplyr)
```

Comment:
The rationale for including the `library` calls is so that it will throw an _error_ if the package is not successfully installed, is that correct?

Reply:
Yes, that is correct.